venia-integration-tests/src/fixtures/wishlist/multipleWishlist/wishlistPageConfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "data": {
     "storeConfig": {
-      "id": 1,
+      "store_code": 1,
```

Comment:
`store_code` is usually a string like "default" since it refers to the Store View Code in the admin.
For clarity, could we update the tests to reflect that?